CODEOWNERS

```diff
@@ -12,6 +12,14 @@
 /java/kotlin-extractor/ @github/codeql-kotlin
 /java/ql/test-kotlin1/ @github/codeql-kotlin
 /java/ql/test-kotlin2/ @github/codeql-kotlin
+/rust/extractor/ @github/codeql-rust-extractor
```

Comment:
Re: subteams, I got the request to have narrower teams to prevent notification overdrive, see [slack](https://github.slack.com/archives/C07GVB2FFAP/p1726158265665559?thread_ts=1726155426.224219&cid=C07GVB2FFAP). But we can reevaluate.

Re: team not existing, I think I did not follow the playbook from the internal process that should automate this.